base/gfspapp/download_server.go

```diff
@@ -18,19 +18,17 @@ var (
 
 var _ gfspserver.GfSpDownloadServiceServer = &GfSpBaseApp{}
 
-func (g *GfSpBaseApp) GfSpDownloadObject(
-	ctx context.Context,
-	req *gfspserver.GfSpDownloadObjectRequest) (
+func (g *GfSpBaseApp) GfSpDownloadObject(ctx context.Context, req *gfspserver.GfSpDownloadObjectRequest) (
 	*gfspserver.GfSpDownloadObjectResponse, error) {
-	task := req.GetDownLoadTask()
+	task := req.GetDownloadObjectTask()
```

Comment:
the task var name is the same as imported package name, maybe use a new name.

Reply:
Updated.